.github/workflows/prerelease.yml

```diff
@@ -183,80 +163,116 @@ jobs:
       - name: Build windows binary
         id: package
         shell: bash
         run : |
-          make build-${{ steps.vars.outputs.NAME }}
-      - name : Create MSI installer
-        shell: pwsh
-        run: |
-          .\scripts\win_msi_build.ps1 -arch ${{ matrix.goarch }} -exporterName ${{ steps.vars.outputs.NAME }} -version ${{ steps.vars.outputs.VERSION }} -exporterGUID ${{ steps.vars.outputs.EXPORTER_GUID }} -upgradeGUID ${{ steps.vars.outputs.UPGRADE_GUID }} -licenseGUID ${{ steps.vars.outputs.LICENSE_GUID }} -configGUID ${{ steps.vars.outputs.CONFIG_GUID }} -pfx_passphrase "$env:OHAI_PFX_PASSPHRASE"
+          make package-${{ steps.vars.outputs.NAME }}
+
       - name: Test win packages installation
         uses: newrelic/integrations-pkg-test-action/windows@v1
         with:
           tag: ${{ steps.vars.outputs.VERSION  }}
           integration: nri-${{ steps.vars.outputs.NAME }}
-          arch: ${{ matrix.goarch }}
+          arch: amd64
           pkgDir: ${{ github.workspace }}\src\repo\exporters\${{ steps.vars.outputs.NAME }}\target\packages
-          pkgName: ${{ steps.vars.outputs.PACKAGE_NAME }}-${{ matrix.goarch }}.${{ steps.vars.outputs.VERSION }}.msi
+          pkgName: ${{ steps.vars.outputs.PACKAGE_NAME }}-amd64.${{ steps.vars.outputs.VERSION }}.msi
           upgrade: false
-      - name: Upload Release Asset MSI
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Upload assets to be released
+        uses: actions/upload-artifact@v3
+        with:
+          name: windows-artifacts
+          retention-days: 1
+          path: |
+            ${{ github.workspace }}\src\repo\exporters\${{ steps.vars.outputs.NAME }}\target\packages\*.msi
+            ${{ github.workspace }}\src\repo\exporters\${{ steps.vars.outputs.NAME }}\target\packages\*.zip
+
+  create_publish_schema:
+    name: Create the publish schema
+    runs-on: ubuntu-latest
+    if: ${{ needs.create_prerelease.outputs.CREATE_RELEASE == 'true' }}
+    needs: [ create_prerelease ]
+    steps:
+      - uses: actions/checkout@v3
+      - run: make create-publish-schema-${{ needs.create_prerelease.outputs.INTEGRATION_NAME }}
+      - uses: actions/upload-artifact@v3
         with:
-          upload_url: ${{ needs.create_prerelease.outputs.UPLOAD_URL }}
-          asset_path: ${{ github.workspace }}\src\repo\exporters\${{ steps.vars.outputs.NAME }}\target\packages\${{ steps.vars.outputs.PACKAGE_NAME }}-${{ matrix.goarch }}.${{ steps.vars.outputs.VERSION }}.msi
-          asset_name: ${{ steps.vars.outputs.PACKAGE_NAME }}-${{ matrix.goarch }}.${{ steps.vars.outputs.VERSION }}.msi
-          asset_content_type: msi package
+          name: publish-schema
+          retention-days: 1
+          path: |
+            scripts/pkg/s3-publish-schema-tmp.yml
 
   publish-to-s3:
     name: Send release assets to S3
```

Comment:
Now it creates the Github release and sends the assets to S3, right? I guess we could rename this as well.

Reply:
Solved in [#a5426dd](https://github.com/newrelic/newrelic-prometheus-exporters-packages/pull/147/commits/a5426ddf7aae5d46b6419d29785cdb4550bab47a)